console.go

```diff
@@ -315,6 +317,17 @@ var consoleCommands = map[string]func(c *console, updateSel updateSelectionFn, a
 			return nil, errArgumentNumber
 		}
 	},
+	"fit_inserting": func(c *console, updateSel updateSelectionFn, args []float32) ([][]float32, error) {
```

Comment:
Coudl you provide an example of valid `fit_inserting` command? Is it something like `fit_inserting 0 1 2` or `fit_inserting 0 1 2 4`?

Reply:
Added to README